cytosnake/utils/config_utils.py

```diff
@@ -41,6 +39,19 @@ def load_configs(config_path: str | Path) -> dict:
     return loaded_configs
 
 
+def load_general_configs() -> dict:
+    """Loads cytosnake's general configurations
+
+    Returns:
+    -------
+    dict
+        dictionary containing the cytosnake general configs
+    """
+    config_dir_path = cp.get_config_dir_path() / "configuration.yaml"
```

Comment:
if you change the name of the this file above, also make sure to propagate the change here.

Reply:
Correct. Any name changes that occurs will also have to be reflected here. Still thinking on ideas on how to prevent harcoding file names. I am open to any suggestions

Reply:
i think it is ok to hardcode some things - ultimately, they will need to be hardcoded somewhere!

You may consider having a single function where these names are hardcoded, and calling the function wherever hardcodes are used. Just don't update the corresponding dictionary keys!

I think what you have now is fine.

Reply:
I was thinking of making a `class` the handles all the file pathing function calls. It's an idea.